### DISCUSSION TITLE
Typescript Dev Server - Hot Reaload Replacement

### DISCUSSION THREAD

**Discussion by Yuri-Lima** [vendurehq/vendure#2822] [General]
Why Vendure in dev mode decided to use `ts-node-dev` over `webpack with ts-loader` ?

Both ones are very powerful but i think for much more control and for large project webpack with ts-loader would be much more helpful. 

Is there anyone here how had a chance to convert it to use webpack with hot reload?

**Comment by michaelbromley** [comment]
Hi,

I would be open to check out a proof-of-concept and if it is a better dev experience we could update the default setup.
